tsl/test/shared/sql/decompress_tracking.sql

```diff
@@ -0,0 +1,26 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+
+CREATE TABLE decompress_tracking(time timestamptz not null, device text, value float, primary key(time, device));
+SELECT table_name FROM create_hypertable('decompress_tracking','time');
+ALTER TABLE decompress_tracking SET (timescaledb.compress, timescaledb.compress_segmentby='device');
+
+INSERT INTO decompress_tracking SELECT '2020-01-01'::timestamptz + format('%s hour', g)::interval, 'd1', random() FROM generate_series(1,10) g;
+INSERT INTO decompress_tracking SELECT '2020-01-01'::timestamptz + format('%s hour', g)::interval, 'd2', random() FROM generate_series(1,20) g;
+INSERT INTO decompress_tracking SELECT '2020-01-01'::timestamptz + format('%s hour', g)::interval, 'd3', random() FROM generate_series(1,30) g;
+
+SELECT count(compress_chunk(ch)) FROM show_chunks('decompress_tracking') ch;
+
+\set VERBOSITY verbose
+
+-- BEGIN; EXPLAIN (analyze,costs off,timing off,summary off) UPDATE decompress_tracking SET value = value + 3 WHERE device = 'd1'; ROLLBACK;
```

Comment:
Do you plan to add or remove these two commented out queries from the tests?

Reply:
I will uncomment but currently they bug out due to https://github.com/timescale/timescaledb/issues/6175